goalgorilla/drupal_social
# DS-709 by maikel-koopman: update jade to pug

How to test:

- remove the folder node_modules in the theme
- run npm install from theme folder
- run gulp task (default)
- make a change in a pug file, check in the browser the change is reflected
- revert your change (if needed)

- [ ] watch that everything works as expected, no errors in your CLI
